politeiawww/proposals.go

```diff
@@ -1752,7 +1718,13 @@ func (p *politeiawww) processEditProposal(ep www.EditProposal, u *user.User) (*w
 	if err != nil {
 		return nil, err
 	}
-	if !filesAreDifferent(cachedProp.Files, ep.Files) {
+	// Check if there were changes in the proposal by comparing
+	// their merkle roots.
```

Comment:
Can we include in this comment the reason why we compare merkle roots if because it captures any changes that were made to either the files or the metadata.